package.json

```diff
@@ -1,25 +1,36 @@
 {
-  "name": "hetmech-frontend",
-  "version": "0.1.0",
-  "private": true,
-  "dependencies": {
-    "react": "^16.7.0",
-    "react-dom": "^16.7.0",
-    "react-scripts": "2.1.3"
-  },
-  "scripts": {
-    "start": "react-scripts start",
```

Comment:
Can you add the following `scripts` section to maintain the compatibility with `create-react-app` commands?
```
  "scripts": {
    "start": "npx webpack-dev-server",
    "build": "npx webpack",
    "test": "npx test",
  },
```
I think it is a good idea to turn off the watch mode in `npx webpack` command. I expected that the building process should go back to the terminal prompt when it is done.

Reply:
I would hold off on these comments for now because we might end up just importing the `react-scripts` scripts anyway. If we do decide to do a "from-scratch" template I will incorporate this comment.

Reply:
Sounds good. Thanks!